coalib/results/AbsolutePosition.py

```diff
@@ -27,6 +27,90 @@ def __init__(self,
     def position(self):
         return self._position
 
+    def __le__(self, other):
+        """
+        Test whether ``self`` is behind or equals the other
+        ``AbsolutePosition``.
+
+        :param other: ``AbsolutePosition`` to compare with.
+        :return:      Whether this ``AbsolutePosition`` is behind the other
+                      one or the same.
+        """
+        if not isinstance(other, self.__class__):
+            return False
+
+        if self._text != other._text:
+            return False
+
+        return self.position <= other.position
```

Comment:
I think you could do something like `return self.position <= other.position and self._text == other._text and isinstance(other, self__class__)`